kqueen_ui/blueprints/ui/templates/ui/organization_manage.html

```diff
@@ -14,20 +14,20 @@ <h3>Members</h3>
   </div>
   <table class="table table-hover">
     <thead>
-      <th>Name</th>
-      <th>E-mail</th>
-      <th>Role</th>
-      <th>Created</th>
+      <th class="col-md-5">Name</th>
+      <th class="col-md-5">E-mail</th>
+      <th class="col-md-3">Role</th>
+      <th class="col-md-4">Created</th>
```

Comment:
AFAIK sum of all `col-md-*` should equal to 12. Here is 17, and last 2 columns don't even have any...